From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chen Chen <cncn@google.com>
Date: Tue, 5 Oct 2021 17:46:02 -0700
Subject: [PATCH] BluetoothSecurity: Add BLUETOOTH_PRIVILEGED permission for
 pairing dialog

Bug: 194300867
Test: Build

Change-Id: I5a496df50550ea7ee6986c960e28ae3e4a056b4b
(cherry picked from commit aebca7f35bf6bf975eeeb443fa8bbeb5d477d642)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 48713066699..86fc53bb18d 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -2098,6 +2098,7 @@
 
         <activity android:name=".bluetooth.BluetoothPairingDialog"
                   android:label="@string/bluetooth_pairing_request"
+                  android:permission="android.permission.BLUETOOTH_PRIVILEGED"
                   android:excludeFromRecents="true"
                   android:windowSoftInputMode="stateVisible|adjustResize"
                   android:theme="@*android:style/Theme.DeviceDefault.Light.Dialog.Alert">
